Scripts/RemediationScripts/Remediate-ConfigureConditionalAccessPolicyForPIM.ps1

```diff
@@ -158,7 +162,7 @@ function Configure-ConditionalAccessPolicyForPIM
 
         .DESCRIPTION
         Remediates 'Azure_Subscription_Configure_Conditional_Access_for_PIM' Control.
```

Comment:
Please update the control id to Azure_Subscription_AuthZ_Configure_Conditional_Access_for_PIM

Reply:
Updated as suggested.